gridfs/grid_file.py

```diff
@@ -421,7 +421,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
         return False
 
 
-class GridOut(object):
+class GridOut(io.IOBase):
```

Comment:
In order to subclass io.IOBase we need to resolve one more compatibility issue. The problem is that `IOBase.__iter__` yields _lines_ in the file while `GridOut.__iter__`(GridOutIterator) yields the entire chunk. In order to support subclassing IOBase we'll need to change GridOutIterator to return lines, delimitated by `b'\n'`.

Reply:
Replied on jira to discuss approach to __iter__ :ok_hand:

Reply:
See comment below